### ISSUE TITLE
Reduce burden on third party!

### ISSUE THREAD

**Issue by DukesMC** [playerrealms/PlayerRealms#2]
Please revive the third party. Doing so will reduce the burden on the host server.

**Comment by RaMenPower** [comment]
I agree. Maybe ThirdParty can reduce administers's burden, can't it?

**Comment by Dnthy** [comment]
I think so too. ThirdParty will be able to reduce the burden on the host server.
